tests/helpers/templates/wheel/standard_tests.py

```diff
@@ -15,7 +15,7 @@ def get_files(**kwargs):
         if str(f.path) == 'LICENSE.txt':
             files.append(File(Path(metadata_directory, 'licenses', f.path), f.contents))
 
-        if f.path.parts[0] not in (kwargs['package_name'], 'tests'):
+        if f.path.parts[0] not in {kwargs['package_name'], 'tests'}:
```

Comment:
It's a cool lint but I think it actively harms performance. In the vast vast vast majority of cases, membership checks are much faster against a tuple literal vs a set literal.

1. A tuple literal is often optimizable to a global singleton and so has 0 construction cost. A set literal is quite expensive to construct every time.

2. Regardless of it being a literal, linearly checking against <5 members is faster than hashing the item and then checking against it in a hashmap.

Reply:
https://github.com/astral-sh/ruff/issues/8758

Reply:
I'm glad someone else thinks the way I do! Do think I should revert this particular one? I would like to.

Reply:
Yeah I think just don't apply this lint and move on. There are many questionable lints 😃

Reply:
Will do! Which others are you not fond of?

Reply:
You're doing great by maintaining your own allowlist of lints :smile:  so I have nothing to add
https://github.com/pypa/hatch/blob/977f88285e9747a0962f7a068b92b356043ff88d/ruff.toml

Here's my allowlist
https://github.com/oprypin/mkdocs-literate-nav/blob/95194eb34bd0eb6fcd4432329be7decd4163305b/pyproject.toml#L106

I don't remember which rules I didn't like. When I didn't like them, I just didn't add them, that's it.

Reply:
@oprypin it doesn’t harm performance, see https://github.com/astral-sh/ruff/issues/8758

Reply:
@flying-sheep but this is a case where the optimizer doesn't kick in